egs/wsj/s5/utils/format_lm_sri.sh

```diff
@@ -104,6 +104,27 @@ awk '{print $1}' $out_dir/words.txt > $tmpdir/voc || exit 1;
 change-lm-vocab -vocab $tmpdir/voc -lm $tmpdir/lm.gz -write-lm $tmpdir/out_lm \
   $srilm_opts || exit 1;
 
+# change-lm-vocab introduces error in out_lm when lm.gz has an order
```

Comment:
What I can do is ignore highest-order n-grams down until there is an order of non-zero cardinality. I think this is what actually happens here. This could be useful as it can potentially interfere with the selection of packed vs regular history keys. Packed keys support up to 4-grams, inclusively. If there is a 0 count 5-gram section, that will pessimize the compile.